gitlab_cli_tool/cli_api.py

```diff
@@ -146,6 +149,76 @@ def get_filtered_data(self):
         elif self.property_name == PropertyName.PIPELINE.value:
             return self.api.run_pipeline(self.branch, self.project_id, self.variables)
 
+    def valid_retag_params(self) -> bool:
+        """
+        Checking if users put correct retag params
+        EXAMPLE:
+        runners retag old1:new1,old2:new2
+        :return: BOOL
+        """
+        if not (self.action[0] == Actions.RETAG.value):
+            return False
+        pairs = self.action[1].split(',')
+        for pair in pairs:
+            if not self.correct_retag_pair(pair):
+                return False
+        return True
+
+    @staticmethod
+    def correct_retag_pair(pair):
+        return len(pair.split(':')) == 2
+
+    def get_tags_to_change(self):
+        tags_to_change = self.action[1].split(',')
+        return [pair.split(':') for pair in tags_to_change]
+
+    def retag_runners(self, runners):
+        if not self.valid_retag_params():
+            raise RuntimeError("Wrong retag arguments. HINT: runners retag old1:new1,old2:new2 ...")
+        tags_to_change = self.get_tags_to_change()
+        runners_after_changes = []
+        for runner in runners:
+            changed, new_runner = self.retag_algorithm(runner, tags_to_change)
+            runners_after_changes.append((changed, runner, new_runner))
+        self.inform_user_about_changes(runners_after_changes)
+        if self.ask_for_change():
+            self.commit_changes_to_runners([runner[2] for runner in runners_after_changes if runner[0]])
+        return [runner[2] for runner in runners_after_changes]
+
+    def inform_user_about_changes(self, runners_after_changes):
+        for runner_info in runners_after_changes:
+            if runner_info[0]:
+                print(runner_info[1]['description'], 'changes: ', runner_info[1]['tag_list'], ' -> ',
+                      runner_info[2]['tag_list'])
+            else:
+                print(runner_info[1]['description'], "can't change")
```

Comment:
```suggestion
        for changed, runner, new_runner in runners_after_changes:
            if changed:
                print(runner['description'], 'changes: ', runner['tag_list'], ' -> ',
                      new_runner['tag_list'])
            else:
                print(runner['description'], "can't change")
```

This would make it clearer I think.

Reply:
Agree!